### PR TITLE
Fix/86104 mu resource url

### PR DESCRIPTION
**Pull request by jesseeproductions**
_Ref:_ [C#86104](https://central.tri.be/issues/86104)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [status.php:181](https://github.com/moderntribe/the-events-calendar/blob/6498ae729f4e950858d85339398e38c398e22de6/src/admin-views/aggregator/status.php#L181)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [status.php:208](https://github.com/moderntribe/the-events-calendar/blob/6498ae729f4e950858d85339398e38c398e22de6/src/admin-views/aggregator/status.php#L208)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [status.php:230](https://github.com/moderntribe/the-events-calendar/blob/6498ae729f4e950858d85339398e38c398e22de6/src/admin-views/aggregator/status.php#L230)

> via jenkins/codesniffer-phpcs

File: src/admin-views/aggregator/status.php [hunk 1]
```diff
@@ -178,7 +178,7 @@
 		?>
         <tr>
             <td class="label">
-                <img src="<?php echo tribe_events_resource_url( 'images/aggregator/facebook.png' ); ?>" /><span><?php esc_html_e( 'Facebook', 'the-events-calendar' ); ?></span>
+                <img src="<?php echo tribe_resource_url( 'images/aggregator/facebook.png' ); ?>" /><span><?php esc_html_e( 'Facebook', 'the-events-calendar' ); ?></span>
```
Comment (juanfra) [inline src/admin-views/aggregator/status.php]: Let's wrap it with `esc_url`

File: src/admin-views/aggregator/status.php [hunk 2]
```diff
@@ -205,7 +205,7 @@
 		?>
         <tr>
             <td class="label">
-                <img src="<?php echo tribe_events_resource_url( 'images/aggregator/eventbrite.png' ); ?>" /><span><?php esc_html_e( 'Eventbrite', 'the-events-calendar' ); ?></span>
+                <img src="<?php echo tribe_resource_url( 'images/aggregator/eventbrite.png' ); ?>" /><span><?php esc_html_e( 'Eventbrite', 'the-events-calendar' ); ?></span>
```
Comment (juanfra) [inline src/admin-views/aggregator/status.php]: Let's wrap it with `esc_url`

File: src/admin-views/aggregator/status.php [hunk 3]
```diff
@@ -227,7 +227,7 @@
 		?>
         <tr>
             <td class="label">
-                <img src="<?php echo tribe_events_resource_url( 'images/aggregator/meetup.png' ); ?>" /><span><?php esc_html_e( 'Meetup', 'the-events-calendar' ); ?></span>
+                <img src="<?php echo tribe_resource_url( 'images/aggregator/meetup.png' ); ?>" /><span><?php esc_html_e( 'Meetup', 'the-events-calendar' ); ?></span>
```
Comment (juanfra) [inline src/admin-views/aggregator/status.php]: Let's wrap it with `esc_url`

File: src/functions/template-tags/general.php [hunk 4]
```diff
@@ -592,7 +592,7 @@ function tribe_events_before_html() {
 		$before = wpautop( $before );
 		$before = do_shortcode( stripslashes( shortcode_unautop( $before ) ) );
 		$before = '<div class="tribe-events-before-html">' . $before . '</div>';
-		$before = $before . '<span class="tribe-events-ajax-loading"><img class="tribe-events-spinner-medium" src="' . tribe_events_resource_url( 'images/tribe-loading.gif' ) . '" alt="' . sprintf( esc_html__( 'Loading %s', 'the-events-calendar' ), $events_label_plural ) . '" /></span>';
+		$before = $before . '<span class="tribe-events-ajax-loading"><img class="tribe-events-spinner-medium" src="' . tribe_resource_url( 'images/tribe-loading.gif' ) . '" alt="' . sprintf( esc_html__( 'Loading %s', 'the-events-calendar' ), $events_label_plural ) . '" /></span>';
```
Comment (juanfra) [inline src/functions/template-tags/general.php]: Let's wrap the src with `esc_url` and then the alt text with `esc_attr__`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Space found before comma in function call | [Migrate.php:312](https://github.com/moderntribe/the-events-calendar/blob/d7be7b5fccdf7093eea4ae6b3a78d881004c8ffe/src/Tribe/Aggregator/Migrate.php#L312)
:x: | Expected 1 space after comma in function call; 2 found | [Migrate.php:313](https://github.com/moderntribe/the-events-calendar/blob/d7be7b5fccdf7093eea4ae6b3a78d881004c8ffe/src/Tribe/Aggregator/Migrate.php#L313)
:warning: | Comment refers to a TODO task | [New.php:611](https://github.com/moderntribe/the-events-calendar/blob/d7be7b5fccdf7093eea4ae6b3a78d881004c8ffe/src/Tribe/Aggregator/Tabs/New.php#L611)

> via jenkins/codesniffer-phpcs

**Pull request merged**